portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Kathleen's Photo Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <style>
+        h1 {text-align: center; font-family: Georgia, serif;}
+        .center {text-align: center;}
+        div.gallery {border: 1px solid #ccc;}
+        div.gallery:hover {border: 1px solid #777;}
+        div.gallery img {width: 100%; height: auto;}
+        div.desc {padding: 15px; text-align: center;}
+        * {box-sizing: border-box;}
+        .responsive {
+            padding: 0 6px;
+            float: left;
+            width: 49.99999%;
+        }
+        @media only screen and (max-width: 700px) {
+            .responsive {
+            width: 49.99999%;
+            margin: 6px 0;
+            }
+        }
+        @media only screen and (max-width: 500px) {
+            .responsive {width: 100%;}
+        }
+        .clearfix:after {content: ""; display: table; clear: both;}
+    </style>
+  </head>
+  <body>
+    <div id="content">
+      <h1 style="background-color:LightGoldenRodYellow; color:DarkGoldenRod">Kathleen's Photo Gallery</h1>
+      <div class="responsive">
+        <div class="gallery">
+            <a target="_blank" href="images/selfie.jpg">
+            <img src="images/selfie.jpg" alt="Selfie" width="600" height="400">
```

Comment:
to avoid repetition, you can apply the style width, and height to all images by including this in your css file:

img. {
width:40px;
height:40px;
}

Reply:
Great idea to mention it for me~ I have changed it.